.gitignore

```diff
@@ -1,3 +1,4 @@
 .gradle/
 build/
-gradle/
\ No newline at end of file
+gradle/
+.idea/
```

Comment:
You should not exclude the .idea directory:
https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
```suggestion
```